service/capabilityService.ts

```diff
@@ -0,0 +1,38 @@
+import { Job } from "../model/job";
+import { Responsibility } from "../model/responsibility";
+import { Capability } from "../model/capability";
+
+const axios = require("axios");
+const capabilityValidator = require("../Validator/capabilityValidator");
+
+
+module.exports.getAllCapabilities = async function (): Promise<Capability> {
+    try {
+        const response = await axios.get("http://localhost:8080/api/capability");
```

Comment:
This url will need to be changed to use the env API_URL or it wont work with AWS backend.